diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6a4951026..e3de65d28 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -317,9 +317,7 @@ if(WIN32)
   target_compile_definitions(srtp3 PUBLIC _CRT_SECURE_NO_WARNINGS)
 endif()
 
-install(TARGETS srtp3 DESTINATION lib
-  EXPORT libSRTPTargets
-)
+install(TARGETS srtp3 EXPORT libSRTPTargets)
 
 install(FILES include/srtp.h crypto/include/auth.h
   crypto/include/cipher.h
